backend/lcfs/web/api/compliance_report/views.py

```diff
@@ -80,12 +82,16 @@ async def get_compliance_report_by_id(
     report_id: int,
     service: ComplianceReportServices = Depends(),
     validate: ComplianceReportValidation = Depends(),
-) -> ComplianceReportBaseSchema:
+) -> ChainedComplianceReportSchema:
     await validate.validate_organization_access(report_id)
 
     mask_statuses = not user_has_roles(request.user, [RoleEnum.GOVERNMENT])
 
-    return await service.get_compliance_report_by_id(report_id, mask_statuses)
+    result = await service.get_compliance_report_by_id(
```

Comment:
`service.get_compliance_report_by_id` this function is being called from backend/lcfs/web/api/organization/views.py as well, may need to make changes in it as well.